sdk/src/main/java/com/horizen/account/state/EvmMessageProcessor.java

```diff
@@ -32,11 +32,15 @@ public byte[] process(Message msg, BaseAccountStateView view, GasPool gas, Block
             throws ExecutionFailedException {
         // prepare context
         var context = new EvmContext();
+        // TODO: set correct ChainID
```

Comment:
I this TODO relevant?
I think we setup it properly in all the places now.

Reply:
Oops, yes. Outdated comment.

Reply:
done